apps/frontend/core-dapp/src/features/withdraw/WithdrawPage.tsx

```diff
@@ -23,9 +23,9 @@ const WithdrawPage: React.FC = () => {
         value={withdrawalAmountInEth}
         showBalance
       />
-      <WithdrawButton />
       <WithdrawWarning />
       <WithdrawSummary />
+      <WithdrawButton />
```

Comment:
Could we rearrange warning to below of button please

```suggestion
      <WithdrawSummary />
      <WithdrawButton />
      <WithdrawWarning />
```

Reply:
That wouldn't be consistent with open trade and deposit:

<img width="1031" alt="Screenshot 2023-05-02 at 09 19 33" src="https://user-images.githubusercontent.com/11708035/235603965-a2b85f1f-0108-493b-9451-aecf30f163fc.png">

Should we do the same on those?

Reply:
sry missed those, but all deposit limit should now be below buttons, but feel free to change this along with other features in separate PRs